src/main.js

```diff
@@ -81,37 +81,43 @@ function displayRandomHomePage() {
   displayRandomTitle();
   displayRandomTagline1();
   displayRandomTagline2();
```

Comment:
Some of these functions seem a little over redundant. I would recommend making your randomizer function a little more flexible so that it works on all arrays. This way, you could call that one function for each cover image, descriptor, and title to create a new poster. This will help condense 3 - 5 functions down into one.